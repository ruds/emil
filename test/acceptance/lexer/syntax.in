( )
(--)
[1, 2, 3]
foo ; bar
foo;bar
{ }
foo.bar
foo . bar ;
...
.
(*xx*)
