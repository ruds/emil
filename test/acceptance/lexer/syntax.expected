0001 LPAREN (
0001 RPAREN )
0002 LPAREN (
0002 ID_OP 2D2D --
0002 RPAREN )
0003 LBRACKET [
0003 ILITERAL 1 1
0003 COMMA ,
0003 ILITERAL 2 2
0003 COMMA ,
0003 ILITERAL 3 3
0003 RBRACKET ]
0004 ID_WORD 666F6F foo
0004 SEMICOLON ;
0004 ID_WORD 626172 bar
0005 ID_WORD 666F6F foo
0005 SEMICOLON ;
0005 ID_WORD 626172 bar
0006 LBRACE {
0006 RBRACE }
0007 QUAL_ID_WORD 666F6F.626172 foo.bar
0008 QUAL_ID_WORD 666F6F.626172 foo . bar
0008 SEMICOLON ;
0009 ELLIPSIS ...
0010 ERROR
0012 END_OF_FILE
